src/systems/kinetic_energy_monitor/KineticEnergyMonitor.hh

```diff
@@ -114,6 +115,10 @@ namespace systems
                            EntityComponentManager &_ecm,
                            EventManager &_eventMgr) final;
 
+    /// Documentation inherited
+    public: void Update(const UpdateInfo &_info,
```

Comment:
Use of `Update` is discouraged. Only the physics system should use it. Can the functionality be added `PreUpdate`?

Reply:
Moved to https://github.com/osrf/subt/pull/858